PROJECT_GOVERNANCE.md

```diff
@@ -0,0 +1,163 @@
+# Tinted Theming Project Governance
+
+Welcome to the Tinted Theming project! This document outlines the
+governance structure and procedures for maintaining and contributing to
+this theming driven community.
+
+## 1. Project Governance and Final Decision Authority
+
+The Tinted Theming project is currently governed by three admins:
+
+- [@balek], who has been actively involved in the community since the
+  original base16 theming project.
+- [@JamyGolden], who has been actively involved in the formation of
+  base16-project and tinted-theming.
+- [@FredHappyface], who recently joined the administration team.
+
+**Final Say:** The three admins collectively have the final say on all
+matters related to the overall direction, policies, and major decisions
+concerning the project. This ensures that the project remains true to
+its core values and goals while also allowing for flexibility and
+community input.
+
+Project-wide decisions are typically made through consensus among the
+admins. In cases where consensus cannot be reached, a majority vote
+among the admins will determine the final decision.
```

Comment:
This is good to call out - I appreciate this. We've had decisions in the past where we sort of deadlocked because we couldn't reach consensus, and it's good to avoid that when possible.